Package.swift

```diff
@@ -37,18 +37,19 @@ let s2nExcludes = ["bin", "codebuild", "coverage", "docker-images",
                    "scripts/", "codebuild", "bindings/rust", "VERSIONING.rst", "tests",
                    "cmake/s2n-config.cmake", "CMakeLists.txt", "README.md", "cmake", "NOTICE", "LICENSE"]
 packageTargets.append(.target(
-    name: "S2N",
+    name: "AwsCS2N",
     dependencies: ["LibCrypto"],
     path: "aws-common-runtime/s2n",
     exclude: s2nExcludes,
     publicHeadersPath: "api",
     cSettings: [
         .headerSearchPath("./"),
         .define("POSIX_C_SOURCE=200809L"),
-        .define("S2N_NO_PQ")
+        .define("S2N_NO_PQ"),
+        .unsafeFlags(["-fno-implicit-module-maps"])
```

Comment:
I think using `unsafeFlags()` in a target definition prohibits the package from being published as a version (it may still be published as a Git SHA, branch, etc.): https://developer.apple.com/documentation/packagedescription/swiftsetting/unsafeflags(_:_:) (see the Discussion on this page.)